tt_metal/hw/inc/dataflow_api.h

```diff
@@ -1151,6 +1151,11 @@ uint32_t get_semaphore(uint32_t semaphore_id) {
     return SEMAPHORE_BASE + semaphore_id * sizeof(uint32_t);
 }
 
+FORCE_INLINE
+uint32_t eth_get_semaphore(uint32_t semaphore_id) {
+    return eth_l1_mem::address_map::SEMAPHORE_BASE + semaphore_id * sizeof(uint32_t);
```

Comment:
Almeet pushed a bug fix related to this. See her fix https://github.com/tenstorrent-metal/tt-metal/pull/4820